code/game/gamemodes/cult/cult_items.dm

```diff
@@ -116,19 +116,216 @@
 	armor = list(melee = 50, bullet = 30, laser = 50,energy = 20, bomb = 25, bio = 10, rad = 0)
 	flags_inv = HIDEGLOVES|HIDESHOES|HIDEJUMPSUIT
 
-
 /obj/item/clothing/head/helmet/space/cult
-	name = "nar-sian bruiser's helmet"
-	desc = "A heavily-armored helmet worn by warriors of the Nar-Sian cult. It can withstand hard vacuum."
+	name = "nar-sien hardened helmet"
+	desc = "A heavily-armored helmet worn by warriors of the Nar-Sien cult. It can withstand hard vacuum."
 	icon_state = "cult_helmet"
 	item_state = "cult_helmet"
 	armor = list(melee = 60, bullet = 50, laser = 30,energy = 15, bomb = 30, bio = 30, rad = 30)
 
 /obj/item/clothing/suit/space/cult
-	name = "nar-sian bruiser's armor"
+	name = "nar-sien hardened armor"
 	icon_state = "cult_armor"
 	item_state = "cult_armor"
-	desc = "A heavily-armored exosuit worn by warriors of the Nar-Sian cult. It can withstand hard vacuum."
-	w_class = 3
+	desc = "A heavily-armored exosuit worn by warriors of the Nar-Sien cult. It can withstand hard vacuum."
+	w_class = 2
 	allowed = list(/obj/item/weapon/tome,/obj/item/weapon/melee/cultblade,/obj/item/weapon/tank/internals/)
 	armor = list(melee = 70, bullet = 50, laser = 30,energy = 15, bomb = 30, bio = 30, rad = 30)
+
+/obj/item/clothing/suit/space/cult/pickup(mob/living/user)
+	..()
+	if(!iscultist(user))
+		user << "<span class='cultlarge'>\"I wouldn't advise that.\"</span>"
+		user << "<span class='warning'>An overwhelming sense of nausea overpowers you!</span>"
+		user.Dizzy(30)
+		user.Weaken(5)
```

Comment:
That's probably right, hulks could always use cult swords to break things, but not people. 

I'll see if that change lets it handle clothing too. 

There will always be edge cases for this though, my initial reaction was just to let people use them and since operatives follow a similar system (can't use weapons, but you can wear their shit), none of these items would have serious balance implications in the case of non-cult.